spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+- {layer: 'pois', kind: 'association'}
+- {layer: 'pois', kind: 'aviary'}
+- {layer: 'pois', kind: 'baby_hatch'}
+- {layer: 'pois', kind: 'bakery'}
+- {layer: 'pois', kind: 'beauty'}
+- {layer: 'pois', kind: 'bed_and_breakfast'}
+- {layer: 'pois', kind: 'blood'}
+- {layer: 'pois', kind: 'blood_bank'}
+- {layer: 'pois', kind: 'books'}
+- {layer: 'pois', kind: 'butcher'}
+- {layer: 'pois', kind: 'car'}
+- {layer: 'pois', kind: 'carousel'}
+- {layer: 'pois', kind: 'chalet'}
+- {layer: 'pois', kind: 'childcare'}
+- {layer: 'pois', kind: 'chiropractor'}
+# reserve for kind_details
+- _reserved: {from: 3403, to: 3423}
+- {layer: 'pois', kind: 'clinic'}
+# reserve for kind_details
+- _reserved: {from: 3425, to: 3445}
+- {layer: 'pois', kind: 'clothes'}
+- {layer: 'pois', kind: 'company'}
+- {layer: 'pois', kind: 'computer'}
+- {layer: 'pois', kind: 'consulting'}
+- {layer: 'pois', kind: 'convenience'}
+- {layer: 'pois', kind: 'craft'}
+- {layer: 'pois', kind: 'dentist'}
+# reserve for kind_details
+- _reserved: {from: 3453, to: 3473}
+- {layer: 'pois', kind: 'dispensary'}
+# reserve for kind_details
+- _reserved: {from: 3475, to: 3495}
+- {layer: 'pois', kind: 'doctors'}
+# reserve for kind_details
+- _reserved: {from: 3497, to: 3517}
+- {layer: 'pois', kind: 'educational_institution'}
+- {layer: 'pois', kind: 'employment_agency'}
+- {layer: 'pois', kind: 'estate_agent'}
+- {layer: 'pois', kind: 'fashion'}
+- {layer: 'pois', kind: 'financial'}
+- {layer: 'pois', kind: 'florist'}
+- {layer: 'pois', kind: 'foundation'}
+- {layer: 'pois', kind: 'gift'}
+- {layer: 'pois', kind: 'government'}
+- {layer: 'pois', kind: 'greengrocer'}
+- {layer: 'pois', kind: 'grocery'}
+- {layer: 'pois', kind: 'guest_house'}
+- {layer: 'pois', kind: 'hairdresser'}
+- {layer: 'pois', kind: 'healthcare_centre'}
+# reserve for kind_details
+- _reserved: {from: 3532, to: 3552}
+- {layer: 'pois', kind: 'hifi'}
+- {layer: 'pois', kind: 'hospice'}
+# reserve for kind_details
+- _reserved: {from: 3555, to: 3575}
+- {layer: 'pois', kind: 'hostel'}
+- {layer: 'pois', kind: 'jewelry'}
+- {layer: 'pois', kind: 'karaoke'}
+- {layer: 'pois', kind: 'lawyer'}
+- {layer: 'pois', kind: 'midwife'}
+# reserve for kind_details
+- _reserved: {from: 3581, to: 3601}
+- {layer: 'pois', kind: 'mobile_phone'}
+- {layer: 'pois', kind: 'motorcycle'}
+- {layer: 'pois', kind: 'newsagent'}
+- {layer: 'pois', kind: 'newspaper'}
+- {layer: 'pois', kind: 'ngo'}
+- {layer: 'pois', kind: 'notary'}
+- {layer: 'pois', kind: 'optician'}
+- {layer: 'pois', kind: 'paediatrics'}
+# reserve for kind_details
+- _reserved: {from: 3610, to: 3630}
+- {layer: 'pois', kind: 'optometrist_paediatrics'}
+# reserve for kind_details
+- _reserved: {from: 3632, to: 3652}
+- {layer: 'pois', kind: 'perfumery'}
+- {layer: 'pois', kind: 'petting_zoo'}
+- {layer: 'pois', kind: 'physician'}
+# reserve for kind_details
+- _reserved: {from: 3656, to: 3676}
+- {layer: 'pois', kind: 'physiotherapist_physiotherapy'}
```

Comment:
Rename to just `physiotherapy` (as `physiotherapist` is covered next few lines down).

Reply:
Fixed in cfe0df9.